pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -407,6 +410,16 @@ public static void main(String[] args) throws Exception {
 
         Histogram reportHistogram = null;
 
+        String statsFileName = "perf-consumer-" + System.currentTimeMillis() + ".hgrm";
```

Comment:
This should probably be turned off by default

Reply:
Should it be disabled by default to maintain backwards compatible behaviour before this change? My intention was to make both `produce` and `consume` behave the same way and this functionality is default enabled for `produce`.

Reply:
I agree